packages/configraph/src/resolvers.ts

```diff
@@ -461,9 +461,10 @@ export class ResolverContext {
   dependsOnPathsObj: Record<string, boolean> = {};
   get dependsOnPaths() { return _.keys(this.dependsOnPathsObj); }
 
-  get(nodePath: string): any {
+  get(nodePath: string, opts?: { allowMissing?: boolean }): any {
     const node = this.entity?.getConfigNodeByPath(nodePath);
     if (!node) {
+      if (opts?.allowMissing) return;
```

Comment:
if `allowMissing` option is toggled on, it wont throw an error anymore.  So far this is useful when defining a vendor schema and you have a computed value, and you dont know if the user will pick all of them. I can imagine other cases too.